ansible/roles/master/tasks/main.yml

```diff
@@ -12,29 +14,21 @@
   tags:
     - binary-update
 
-- name: write the config file for the api server
-  template: src=apiserver.j2 dest={{ kube_config_dir }}/apiserver
-  notify:
-    - restart apiserver
+- name: Make sure manifests directory exists
+  file: path={{ kube_manifest_dir }} state=directory mode=0755 
 
 - name: Ensure that a token auth file exists (addons may populate it)
   file: path={{ kube_token_dir }}/known_tokens.csv state=touch
   changed_when: false
 
-- name: add cap_net_bind_service to kube-apiserver
-  capabilities: path=/usr/bin/kube-apiserver capability=cap_net_bind_service=ep state=present
-  when: not is_atomic and not is_coreos
```

Comment:
If this is being removed b/c we are running kube-api in a pod, we may need to add this logic to the template being used for the kube-api pod:

http://osdir.com/ml/scm-fedora-commits/2015-06/msg33074.html